[2025-01-29T10:54:09.589+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-29T10:54:09.663+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.location_Transform manual__2025-01-29T10:53:39.199622+00:00 [queued]>
[2025-01-29T10:54:09.696+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.location_Transform manual__2025-01-29T10:53:39.199622+00:00 [queued]>
[2025-01-29T10:54:09.698+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 3
[2025-01-29T10:54:09.758+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): location_Transform> on 2025-01-29 10:53:39.199622+00:00
[2025-01-29T10:54:10.020+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1483) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-29T10:54:10.058+0000] {standard_task_runner.py:72} INFO - Started process 1504 to run task
[2025-01-29T10:54:10.120+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'location_Transform', 'manual__2025-01-29T10:53:39.199622+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmp1t_5gvj1']
[2025-01-29T10:54:10.159+0000] {standard_task_runner.py:105} INFO - Job 27: Subtask location_Transform
[2025-01-29T10:54:10.763+0000] {task_command.py:467} INFO - Running <TaskInstance: main_dag.location_Transform manual__2025-01-29T10:53:39.199622+00:00 [running]> on host ae52aba2dca2
[2025-01-29T10:54:12.106+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='itsMe!!' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='location_Transform' AIRFLOW_CTX_EXECUTION_DATE='2025-01-29T10:53:39.199622+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-29T10:53:39.199622+00:00'
[2025-01-29T10:54:12.144+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-29T10:54:17.675+0000] {logging_mixin.py:190} INFO - <class 'pandas.core.frame.DataFrame'>
Index: 399 entries, 0 to 2563
Data columns (total 5 columns):
 #   Column       Non-Null Count  Dtype 
---  ------       --------------  ----- 
 0   Postal Code  399 non-null    int64 
 1   Country      399 non-null    object
 2   Region       399 non-null    object
 3   State        399 non-null    object
 4   City         399 non-null    object
dtypes: int64(1), object(4)
memory usage: 18.7+ KB
[2025-01-29T10:54:17.684+0000] {logging_mixin.py:190} INFO - None
[2025-01-29T10:54:18.491+0000] {logging_mixin.py:190} INFO - !_upload_to_S3_complete_!
[2025-01-29T10:54:18.497+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-29T10:54:18.569+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-29T10:54:18.572+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=location_Transform, run_id=manual__2025-01-29T10:53:39.199622+00:00, execution_date=20250129T105339, start_date=20250129T105409, end_date=20250129T105418
[2025-01-29T10:54:18.800+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-29T10:54:18.888+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
